deploy/without_load_balancer/iam_bindings.tf

```diff
@@ -2,156 +2,152 @@
-  members = ["serviceAccount:${var.tag_creator_sa}"]
+  project    = var.tag_engine_project
+  role       = "roles/datacatalog.tagTemplateViewer"
+  members    = ["serviceAccount:${var.tag_creator_sa}"]
   depends_on = [google_project_service.tag_engine_project]
 }
 
 resource "google_project_iam_binding" "viewer" {
-  project = var.tag_engine_project
-  role    = "roles/datacatalog.viewer"
-  members = ["serviceAccount:${var.tag_creator_sa}"]
+  project    = var.tag_engine_project
+  role       = "roles/datacatalog.viewer"
+  members    = ["serviceAccount:${var.tag_creator_sa}"]
   depends_on = [google_project_service.tag_engine_project]
 }
 
 resource "google_project_iam_binding" "dataEditor" {
-  project = var.tag_engine_project
-  role    = "roles/bigquery.dataEditor"
-  members = ["serviceAccount:${var.tag_creator_sa}"]
+  project    = var.tag_engine_project
+  role       = "roles/bigquery.dataEditor"
+  members    = ["serviceAccount:${var.tag_creator_sa}"]
   depends_on = [google_project_service.tag_engine_project]
 }
 
 resource "google_project_iam_binding" "jobUser" {
-  project = var.tag_engine_project
-  role    = "roles/bigquery.jobUser"
-  members = ["serviceAccount:${var.tag_creator_sa}"]
+  project    = var.tag_engine_project
+  role       = "roles/bigquery.jobUser"
+  members    = ["serviceAccount:${var.tag_creator_sa}"]
   depends_on = [google_project_service.tag_engine_project]
 }
-	
+
 resource "google_project_iam_binding" "metadataViewer" {
-  project = var.tag_engine_project
-  role    = "roles/bigquery.metadataViewer"
-  members = ["serviceAccount:${var.tag_creator_sa}"]
+  project    = var.tag_engine_project
+  role       = "roles/bigquery.metadataViewer"
+  members    = ["serviceAccount:${var.tag_creator_sa}"]
   depends_on = [google_project_service.tag_engine_project]
 }
 
 resource "google_project_iam_binding" "loggingViewer" {
-  project = var.tag_engine_project
-  role    = "roles/logging.viewer"
-  members = ["serviceAccount:${var.tag_creator_sa}"]
+  project    = var.tag_engine_project
+  role       = "roles/logging.viewer"
+  members    = ["serviceAccount:${var.tag_creator_sa}"]
   depends_on = [google_project_service.tag_engine_project]
 }
 
 resource "google_project_iam_binding" "PolicyTagReader" {
-  project = var.tag_engine_project
-  role    = "projects/${var.tag_engine_project}/roles/PolicyTagReader"
-  members = ["serviceAccount:${var.tag_creator_sa}"]
-  depends_on = [google_project_service.tag_engine_project]
+  project    = var.tag_engine_project
+  role       = google_project_iam_custom_role.policy_tag_reader.id
+  members    = ["serviceAccount:${var.tag_creator_sa}"]
 }
 
 resource "google_project_iam_binding" "BigQuerySchemaUpdate" {
-  project = var.tag_engine_project
-  role    = "projects/${var.bigquery_project}/roles/BigQuerySchemaUpdate"
-  members = ["serviceAccount:${var.tag_creator_sa}"]
-  depends_on = [google_project_service.tag_engine_project]
+  project    = var.bigquery_project
```

Comment:
@shirleycohen can you confirm that the project IAM binding for `BigQuerySchemaUpdate` should be on the BigQuery Project rather than the tag engine project? The custom role was created on the bq project, but the permission was set on the tag engine project.

Reply:
Confirmed and fixed.